packages/react/src/components/Modal/Modal.base.tsx

```diff
@@ -259,7 +259,7 @@ export const ModalBase: React.FunctionComponent<IModalProps> = React.forwardRef<
     }, [internalState, setModalMenuClose]);
 
     const handleDrag = React.useCallback(
-      (ev: React.MouseEvent<HTMLElement> & React.TouchEvent<HTMLElement>, dragData: IDragData): void => {
+      (_: React.MouseEvent<HTMLElement> & React.TouchEvent<HTMLElement>, dragData: IDragData): void => {
```

Comment:
if this argument is not used at all what's the purpose of its existence ?

is the `handleDrag` reference being exposed as public api ?

Reply:
> if this argument is not used at all what's the purpose of its existence ?

because we need to use the second and cannot completely omit first one, so I just changed it to underscore to indicate it's intentionally ignored.

![image](https://github.com/user-attachments/assets/e9aa9320-3d93-4b45-a593-c639b09c7656) Do you think setting explicitly to any would be better in this case? 

> is the handleDrag reference being exposed as public api ?

It's used inside component implementation logic and not being exported, react.api.md is not changed

Reply:
> Do you think setting explicitly to any would be better in this case?

`unknown` probably , even better to refactor these functions to not have that signature. I have no context about style guide of authoring callback handlers in v8 but it seems to be inconsistent.

good to see  that our changes don't modify any public api though.

Reply:
hm, I think to have explicit type would be better over having any / unknown, especially as in this case, where it was already typed before